qulice-pmd/src/main/java/com/qulice/pmd/Files.java

```diff
@@ -1,73 +0,0 @@
-/**
- * Copyright (c) 2011-2016, Qulice.com
- * All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions
- * are met: 1) Redistributions of source code must retain the above
- * copyright notice, this list of conditions and the following
- * disclaimer. 2) Redistributions in binary form must reproduce the above
- * copyright notice, this list of conditions and the following
- * disclaimer in the documentation and/or other materials provided
- * with the distribution. 3) Neither the name of the Qulice.com nor
- * the names of its contributors may be used to endorse or promote
- * products derived from this software without specific prior written
- * permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
- * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
- * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
- * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
- * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
- * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
- * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
- * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
- * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
- * OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-package com.qulice.pmd;
-
-import com.qulice.spi.Environment;
-import java.io.File;
-import java.util.Collection;
-import java.util.LinkedList;
-import net.sourceforge.pmd.util.datasource.DataSource;
-import net.sourceforge.pmd.util.datasource.FileDataSource;
-
-/**
- * Contains methods to work with source files.
- *
- * @author Dmitry Bashkin (dmitry.bashkin@qulice.com)
- * @version $Id$
- * @since 0.3
- */
-@SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
-public final class Files {
-
-    /**
-     * The environment.
-     */
-    private final Environment environment;
-
-    /**
-     * Constructor.
-     * @param environment The environment.
-     */
-    public Files(final Environment environment) {
-        this.environment = environment;
-    }
-
-    /**
-     * Get full list of files to process.
-     * @return Collection of data sources.
-     */
-    public Collection<DataSource> sources() {
-        final Collection<DataSource> sources = new LinkedList<>();
-        for (final File file : this.environment.files("*.java")) {
```

Comment:
@krzyk indeed... but does it mean that e.g. `pmd-jsp` was not working at all? https://github.com/teamed/qulice/blob/master/qulice-pmd/pom.xml#L81